package.json

```diff
@@ -77,7 +77,7 @@
     "@emotion/jest": "^11.3.0",
     "@emotion/react": "^11.1.5",
     "@homebound/rtl-react-router-utils": "^1.0.3",
-    "@homebound/rtl-utils": "^2.51.0",
+    "@homebound/rtl-utils": "^2.59.3",
```

Comment:
Had to bump for a fix to `rerender` that we made recently, that was otherwise making each `r.rerender` look like a brand new render.